src/linear_algebra/affine_space/barycentric_coords.lean

```diff
@@ -146,6 +146,12 @@ begin
   exact b.coord_apply_combination_of_mem (finset.mem_univ i) hw,
 end
 
+lemma ext_elem [fintype ι] {q₁ q₂ : P} (h : ∀ i, b.coord i q₁ = b.coord i q₂) : q₁ = q₂ :=
```

Comment:
Is there already another `ext` lemma? I get this name, but what do you think of `ext_coord`?

Reply:
There isn't another `ext` lemma (though in time there should be).

I chose this name for consistency with the corresponding naming for linear bases: https://github.com/leanprover-community/mathlib/blob/a779f6cfde79db610d8bfd92d163852c05985caa/src/linear_algebra/basis.lean#L215